### ISSUE TITLE
MerkleValidationError after commit notify

### ISSUE THREAD

**Issue by YuMingLiao** [agentm/project-m36#362]
```
[root@nixos:~/TwitterClone/src]# tutd -d testdb
Warning: Haskell scripting disabled: ScriptSessionLoadError <command line>: cannot satisfy -trust project-m36
    (use -v for more information)
Project:M36 TutorialD Interpreter 0.9.5
Type ":help" for more information.
A full tutorial is available at:
https://github.com/agentm/project-m36/blob/master/docs/tutd_tutorial.markdown
TutorialD (master/main): 
TutorialD (master/main): a := true
TutorialD (master/main): notify a a a a
TutorialD (master/main): :commit
TutorialD (master/main): :quit
Goodbye.

[root@nixos:~/TwitterClone/src]# tutd -d testdb
Warning: Haskell scripting disabled: ScriptSessionLoadError <command line>: cannot satisfy -trust project-m36
    (use -v for more information)
Failed to create database connection: DatabaseValidationError [MerkleValidationError 42986014-4789-410e-a194-efc514ea4a38 (MerkleHash {_unMerkleHash = "\232\203\&4\247\199\177\241ez!t\"L\217,\161\154&\151\173\157#(\150`r\DLE\161\169u\"h"}) (MerkleHash {_unMerkleHash = "\131?\141\235D\190\140_Ut@\194d\217\162\139\172!\ENQ\201\191SO\205\159\242\US\223I\212\199o"})]

```

**Comment by agentm** [comment]
Sorry about this- it looks like the notifications are always read back from disk as empty. Will fix.

**Comment by agentm** [comment]
Fixed in a659f31fc09b244c640002808969d6982135a732.

**Comment by YuMingLiao** [comment]
Tested and working. Thx!
